client/src/research/calculations.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+import {InteractionTypes} from '../shared/data.js';
+
+
+export function rightSwipes(interactions){
+  return interactions.filter(row =>{
+    if (row.interaction.type === InteractionTypes.SWIPE_RIGHT) return true;
+    return false;
+  });
+}
+export function totalSwipes(interactions, key){
+  return _.countBy(interactions, row => row.interaction.turn[key]);
+}
+
+export function percentRightPerProfile(interactions, key){
```

Comment:
This one has a bit of logic, can you add a comment describing what this function does and what it returns?